rs/nervous_system/integration_tests/src/pocket_ic_helpers.rs

```diff
@@ -395,6 +456,13 @@ impl NnsInstaller {
         self
     }
 
+    /// Requests the NNS Governance to be initialized with the following neurons.
+    /// Overrides 'with_neurons_fund_hotkeys'.
```

Comment:
I think overriding can lead to confusion, could we maybe have a precondition that `with_neurons_fund_hotkeys` and `with_neurons` are exclusive?